db/k6.js

```diff
@@ -10,7 +10,7 @@ export const options = {
 }
 
 export default function() {
-  const product = Math.floor(Math.random() * 1000011);
-  http.get(`http://localhost:3000/products/${product}`);
+  const product = Math.floor(Math.random() * 1000);
+  http.get(`http://54.202.83.82:3000/products/1234`);
```

Comment:
this should use .env files so you dont have to recommit each time you want to change what you are testing

Reply:
or maybe not, since this doesnt affect the codebase, just when you are testing